src/iexamine.cpp

```diff
@@ -5247,6 +5252,10 @@ void iexamine::workbench_internal( player &p, const tripoint &examp,
     if( !part ) {
         amenu.addentry( get_items,    items_at_loc,    'g', _( "Get items" ) );
     }
+    if( is_undeployable ) {
+        amenu.addentry( undeploy,     true,            't', string_format( _( "Take down the %s" ),
```

Comment:
`addentry` does string formatting itself, the caller doesn't need it.
```suggestion
        amenu.addentry( undeploy,     true,            't', _( "Take down the %s" ),
```

Reply:
Did not know that. Thanks.